tests/frontend/pages/Packages/packages.py

```diff
@@ -0,0 +1,110 @@
+from random import choice
+from urllib.parse import urljoin
+from tests.frontend.pages.base import Base
+from selenium.webdriver.common.keys import Keys
+
+
+class Packages(Base):
+    def __init__(self, driver, *args, **kwargs):
+        super().__init__(self, *args, **kwargs)
+        self.driver = driver
+        self.endpoint = "/admin/#/packages"
+
+    def load(self):
+        url = urljoin(self.base_url, self.endpoint)
+        self.driver.get(url)
+
+    def check_threebot_deployer_package(self):
```

Comment:
this method name is confusing, we are adding threebot package or checking that it is installed.